clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -578,14 +628,35 @@ private MetadataResponse newMetadataResponse(String topic, Errors error) {
     private Fetcher<byte[], byte[]> createFetcher(int maxPollRecords,
```

Comment:
Would it not be easier to track all the overloads if we had a consistent order for parameters? That is, `SubscriptionState` and `Metrics` would always be the first two parameters.